plugins/api-docs/src/components/AsyncApiDefinitionWidget/AsyncApiDefinition.tsx

```diff
@@ -175,15 +176,24 @@ const config = {
 
 type Props = {
   definition: string;
+  resolvers?: AsyncApiResolver[];
 };
 
-export const AsyncApiDefinition = ({ definition }: Props): JSX.Element => {
+export const AsyncApiDefinition = ({
+  definition,
+  resolvers,
+}: Props): JSX.Element => {
   const classes = useStyles();
   const theme = useTheme();
   const classNames = `${classes.root} ${
     theme.palette.type === 'dark' ? classes.dark : ''
   }`;
 
+  // Overwrite default resolvers if custom ones are set
+  if (resolvers) {
+    config.parserOptions.__unstable.resolver.resolvers = resolvers;
+  }
+
```

Comment:
I would move `config` inside the component to avoid any side effect

```suggestion
  const config = {
    parserOptions: {
      __unstable: {
        resolver: {
          resolvers: resolvers || [httpsFetchResolver, httpFetchResolver],
        },
      },
    },
  };
```